unit-tests/src/test/java/org/eclipse/collections/impl/list/IntervalTest.java

```diff
@@ -467,8 +506,10 @@ public void size()
         Verify.assertSize(12, Interval.fromToBy(1, 100, 9));
         Verify.assertSize(10, Interval.fromToBy(1, 100, 10));
         Verify.assertSize(11, Interval.fromTo(0, 10));
+        Verify.assertSize(11, Interval.fromToExclusive(0, 11));
         Verify.assertSize(1, Interval.zero());
         Verify.assertSize(11, Interval.fromTo(0, -10));
+        Verify.assertSize(11, Interval.fromToExclusive(0, -9));
```

Comment:
This looks incorrect. `Interval.fromTo(0, -10)` has a size of 11 right above. I would expect this is a size of 9.

Reply:
Updated